// @hubspot/ui-extensions/dist/coreComponents.js
import { createRemoteReactComponent } from '@remote-ui/react';
export const Alert = createRemoteReactComponent('Alert');
export const Button = createRemoteReactComponent('Button', {
    fragmentProps: ['overlay'],
});
export const ButtonRow = createRemoteReactComponent('ButtonRow');
export const Card = createRemoteReactComponent('Card');
export const DescriptionList = createRemoteReactComponent('DescriptionList');
export const DescriptionListItem = createRemoteReactComponent('DescriptionListItem');
export const Divider = createRemoteReactComponent('Divider');
export const EmptyState = createRemoteReactComponent('EmptyState');
export const ErrorState = createRemoteReactComponent('ErrorState');
export const Form = createRemoteReactComponent('Form');
export const Heading = createRemoteReactComponent('Heading');
export const Image = createRemoteReactComponent('Image', {
    fragmentProps: ['overlay'],
});
export const Input = createRemoteReactComponent('Input');
export const Link = createRemoteReactComponent('Link', {
    fragmentProps: ['overlay'],
});
export const TextArea = createRemoteReactComponent('TextArea');
// Textarea was changed to TextArea
// Exporting both for backwards compat
/** @deprecated use TextArea instead. With a capital A.*/
export const Textarea = createRemoteReactComponent('Textarea');
export const LoadingSpinner = createRemoteReactComponent('LoadingSpinner');
export const ProgressBar = createRemoteReactComponent('ProgressBar');
export const Select = createRemoteReactComponent('Select');
export const Tag = createRemoteReactComponent('Tag', {
    fragmentProps: ['overlay'],
});
export const Text = createRemoteReactComponent('Text');
export const Tile = createRemoteReactComponent('Tile');
/** @deprecated use Flex instead. It will be removed in the next release. */
export const Stack = createRemoteReactComponent('Stack');
export const ToggleGroup = createRemoteReactComponent('ToggleGroup');
export const StatisticsItem = createRemoteReactComponent('StatisticsItem');
export const Statistics = createRemoteReactComponent('Statistics');
export const StatisticsTrend = createRemoteReactComponent('StatisticsTrend');
export const Table = createRemoteReactComponent('Table');
export const TableFooter = createRemoteReactComponent('TableFooter');
export const TableCell = createRemoteReactComponent('TableCell');
export const TableRow = createRemoteReactComponent('TableRow');
export const TableBody = createRemoteReactComponent('TableBody');
export const TableHeader = createRemoteReactComponent('TableHeader');
export const TableHead = createRemoteReactComponent('TableHead');
export const NumberInput = createRemoteReactComponent('NumberInput');
export const Box = createRemoteReactComponent('Box');
export const StepIndicator = createRemoteReactComponent('StepIndicator');
export const Accordion = createRemoteReactComponent('Accordion');
export const MultiSelect = createRemoteReactComponent('MultiSelect');
export const Flex = createRemoteReactComponent('Flex');
export const DateInput = createRemoteReactComponent('DateInput');
export const Checkbox = createRemoteReactComponent('Checkbox');
export const RadioButton = createRemoteReactComponent('RadioButton');
export const List = createRemoteReactComponent('List');
export const Toggle = createRemoteReactComponent('Toggle');
export const Dropdown = createRemoteReactComponent('Dropdown');
export const Panel = createRemoteReactComponent('Panel');
export const PanelFooter = createRemoteReactComponent('PanelFooter');
export const PanelBody = createRemoteReactComponent('PanelBody');
export const PanelSection = createRemoteReactComponent('PanelSection');
export const StepperInput = createRemoteReactComponent('StepperInput');
export const Modal = createRemoteReactComponent('Modal');
export const ModalBody = createRemoteReactComponent('ModalBody');
export const ModalFooter = createRemoteReactComponent('ModalFooter');
export const Icon = createRemoteReactComponent('Icon');
export const StatusTag = createRemoteReactComponent('StatusTag');
